coordinator/tests/resources/primary-care

```diff
@@ -1 +1 @@
-../../../models/examples/primary-care/
\ No newline at end of file
+../../../examples/primary-care
```

Comment:
Do we still need these symlinked or can they now be referenced from `@models`?

Reply:
can now be referenced from `@models`, but didn't want to make this pr any bigger, but guess that ship sailed a while ago! happy to include it in this change if we want?

Reply:
as discussed, will open a new pr after this one